charmcraft.yaml

```diff
@@ -2,14 +2,15 @@ type: charm
 bases:
```

Comment:
Unfortunately substrate-interface package need the charm to be built on the same series that it will run on. When building on Jammy and run on Focal it could not import `xxhash` which is a dependency for substrate-interface.

I also removed support for 21.10. It's kind of a weird release to support I would say. It's also end of life.